doc/modules/reference.rst

```diff
@@ -277,3 +277,31 @@ uses.
    clean
 
 
+.. _connectivity_ref:
+
+:mod:`nilearn.connectivity`: Functional Connectivity
+====================================================
+
+.. automodule:: nilearn.connectivity
+   :no-members:
+   :no-inherited-members:
+
+**Classes**:
+
+.. currentmodule:: nilearn.connectivity.connectivity_matrices
```

Comment:
I am not the best one to talk about doc generation but if you look at the doc generated by CircleCI, most of sections are empty because the import is not precise enough.